src/main/java/uk/gov/hmcts/et/common/model/ccd/types/SendNotificationType.java

```diff
@@ -59,8 +59,9 @@ public class SendNotificationType {
     private String sendNotificationWhoMadeJudgement;
     @JsonProperty("respondCollection")
     private List<PseResponseTypeItem> respondCollection;
-
     @JsonProperty("respondNotificationTypeCollection")
     private List<GenericTypeItem<RespondNotificationType>> respondNotificationTypeCollection;
+    @JsonProperty("notificationState")
+    private String notificationState;
```

Comment:
Is this part of the sendNotification event? is so it should have sendNotification prefix like the other fields? do we need the a linked field in caseData?

Reply:
as discssed happy to approve

Reply:
It will be always hidden it is used for citizen-ui purposes(visualilsation of statuses for each notification in the list)